bot.py

```diff
@@ -141,22 +145,31 @@ async def get_rank(ctx, *, name: str):  # using * for get a summoner name with s
 
         # Setting variables for summoner information to display as field
         summoner_total_game = summoner_info["solo_win"] + summoner_info["solo_loss"]
-        solo_rank_win_percentage = int(
-            summoner_info["solo_win"] / summoner_total_game * 100
+
+        # Due to zero division error, need to handle situation where total games are zero
+        if summoner_total_game == 0:
+            solo_rank_win_percentage = 0
+        else:
+            solo_rank_win_percentage = int(
+                summoner_info["solo_win"] / summoner_total_game * 100
```

Comment:
I think you can handle this one line - unless you need if/else block for other reason
`solo_rank_win_percentage =  0 if summoner_total_game == 0 else  int(summoner_info["solo_win"] /summoner_total_game * 100`

'true' if True else 'false'